db/migrate/20161128193435_add_researcher_quote_to_project_contents.rb

```diff
@@ -0,0 +1,7 @@
+class AddResearcherQuoteToProjectContents < ActiveRecord::Migration
+  def change
+    add_column :project_contents, :researcher_quote, :text
```

Comment:
This should have a default value for consistency with the other attributes on that table. We should serialize the empty so we are consistent for all clients (not just pfe use cases)